assets/prototype/application/services/apollo/relations/types.ts

```diff
@@ -1,33 +1,51 @@
+import { EspressoDatetime, EspressoPrice, EspressoPriceType, EspressoTicket } from '../../../generated/types';
+
 interface CommonProps {
 	relation?: string;
 	relationId?: string;
 	relationIds?: string[];
 }
+
 export interface RelationFunctionProps extends CommonProps {
 	entity: string;
 	entityId: string;
 }
+
 export interface RelationAction extends CommonProps {
 	type: string;
 	entity?: string;
 	entityId?: string;
 	data?: any;
 }
+
+interface Entity {
+	id: string;
+	dbid: number;
+}
+
+interface DatetimeProps extends Entity, EspressoDatetime {}
+interface PriceProps extends Entity, EspressoPrice {}
+interface PriceTypeProps extends Entity, EspressoPriceType {}
+interface TicketProps extends Entity, EspressoTicket {}
```

Comment:
nice 👌

Reply:
This could be moved to or merged with `assets/prototype/domain/eventEditor/data/types.ts`

Reply:
Also, it's `dbId` not `dbid`.